lib/modules/asset/AssetsGroupsController.ts

```diff
@@ -159,12 +171,60 @@ export class AssetsGroupsController {
     }
   }
 
+  async checkGroupName(
+    engineId: string,
+    body: AssetsGroupsBodyRequest,
+    assetId?: string
+  ) {
+    if (typeof body.name !== "string") {
+      return;
+    }
```

Comment:
If you use the `request.getBodyString` method, you won't need to do those kind of checks